src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+public class Ball {
+    private int number;
+    private String color;
+
+    public int getNumber() {
+        return number;
+    }
+
+    public void setNumber(int number) {
+        this.number = number;
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    public void setColor(String color) {
+        this.color = color;
+    }
+
+    @Override
+    public String toString() {
+        return "Ball's color: " + getColor() + ", and its number: " + getNumber();
+    }
+}

```

Comment:
Remember to specify an access modifier for the class. It's a good practice to define whether a class should be public or package-private explicitly.